certified-connectors/COSMO Bot/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1429 @@
+            "in": "body",
+            "name": "Request body",
+            "schema": {
+              "type": "object",
+              "required": [
+                "userEmail",
+                "queryText",
+                "answerShortDescription",
+                "feedbackText"
+              ],
+              "properties": {
+                "userEmail": {
+                  "title": "User Email",
+                  "type": "string"
+                },
+                "queryText": {
+                  "title": "Question that was asked by the user",
+                  "type": "string"
+                },
+                "answerShortDescription": {
+                  "title": "Short description of the answer",
+                  "type": "string"
+                },
+                "feedbackText": {
+                  "title": "Feedback regarding the answer",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "required": [
+                "ticket"
+              ],
+              "properties": {
+                "ticket": {
+                  "$ref": "#/definitions/Ticket"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/close-ticket": {
+      "post": {
+        "summary": "Close a ticket",
+        "operationId": "CloseTicket",
+        "description": "Closes an open ticket on the connected instance.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "Request body",
+            "schema": {
+              "type": "object",
+              "required": [
+                "ticketId",
+                "editorEmail",
+                "editorComment"
+              ],
+              "properties": {
+                "ticketId": {
+                  "title": "Unique Ticket ID",
+                  "type": "string"
+                },
+                "editorEmail": {
+                  "title": "Email of the user that processed the ticket",
+                  "type": "string"
+                },
+                "editorComment": {
+                  "title": "Comment that will be sent to the user",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
+    "/webhooks/{id}": {
+      "delete": {
+        "summary": "Delete a webhook",
+        "operationId": "DeleteTrigger",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "type": "integer",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@jenkoc Please 'replace' all the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.